### DISCUSSION TITLE
How to reset scrollbars

### DISCUSSION THREAD

**Discussion by barotto** [mikke89/RmlUi#254] [Q&A] [answered]
Hi,
when I set the content of an element with `SetInnerRml()`, its scrollbars are not reset to an initial position (top-left), but remain in the position they are at set from the previous interaction with the content.

I've already tried with
```
element->SetInnerRml(newcontent);
element->GetElementScroll()->FormatScrollbars();
```
but to no avail.

Any suggestions?

**Comment by mikke89** [comment, marked as answer]
Hello! This does sound like a bug to me. Can you give a bit more details, something I can more easily reproduce?

**Reply by barotto** [reply]
I'll try to reproduce with one of the samples.

**Reply by barotto** [reply]
Using the `Sandbox` tab in the `demo` sample, when you update the content with simple text the preview area's scrollbars won't reset although the element's content is updated with:
```
iframe->SetInnerRML(string);
```
But in this particular case I don't really expect the view to change when I update the RML content.
In my case though I need to reset the view.

**Reply by mikke89** [reply]
Not sure I follow exactly what you're trying to do. With reset view, do you mean to scroll to the top-left corner? If so, you can use `Element::SetScrollTop` and `Element::SetScrollLeft`.

**Reply by barotto** [reply]
I appreciate your patience, and understand your confusion, which is caused by my inability to press ctrl-f and search for "scroll" in Element.h ... ouch

**Reply by mikke89** [reply]
Hah, no problem, cheers!
